### DISCUSSION TITLE
Tail-Based Sampling with different policies for different Use Cases

### DISCUSSION THREAD

**Discussion by alexchowle** [open-telemetry/opentelemetry-collector-contrib#16815] [Q&A]
After some research into Tail-Based Sampling in the Collector, I understand that all Spans for a  given Trace need to be handled by the same Collector in order for TBS to be accurate/no fragmented Traces. I also understand the ability to load-balance and pin a Trace to a given Collector via the Load Balancing Exporter.

What I can't work out is if I can have different TBS Policies in the same Collector. For example, Application-A wants TBS setup for 100% of Errors, and 25% of everything else. Application-B has a different TBS Policy in the same Collector config. 

Is that possible to do?

**Comment by jpkrohling** [comment]
You can play around with the "and" sampler: 

1. and sampler with: application=a, err=true
2. and sampler with: application=a, probabilistic=25%
3. and sampler with: application=b, ...
